instrumentation/types.go

```diff
@@ -4,19 +4,10 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/odigos-io/odigos/common"
+	"github.com/odigos-io/odigos/distros/distro"
 	"github.com/odigos-io/odigos/instrumentation/detector"
 )
 
-// OtelDistribution is a customized version of an OpenTelemetry component.
-// see https://opentelemetry.io/docs/concepts/distributions  and https://github.com/odigos-io/odigos/pull/1776#discussion_r1853367917 for more information.
-// TODO: This should be moved to a common root package, since it will require a bigger refactor across multiple components,
-// we use this local definition for now.
-type OtelDistribution struct {
```

Comment:
now the instrumentation module uses the exact same distro as odigos, so no need for duplication